.gitignore

```diff
@@ -1,3 +1,4 @@
+_site
 # ignore .svn metadata files
 .svn
```

Comment:
Just lol

Reply:
@jamezp Should I also ignore CVS files? :D